app/Http/Livewire/Importer.php

```diff
@@ -319,6 +174,287 @@ public function mount()
             'location' =>    trans('general.locations'),
         ];
 
+        // set the variables here so we can translate them!
+        $this->general_fields = [
```

Comment:
I think we'll have a better time if we simply delete this `general_fields` thing completely. A little repetition won't kill us, and it'll be a lot clearer - and I think some code will just disappear. `getColumns()`, for example, will only really make sense for Assets to get its custom fields.